core/new-gui/src/app/workspace/component/operator-panel/operator-label/operator-label.component.html

```diff
@@ -1,3 +1,3 @@
 <mat-card class="texera-operator-label-body mat-elevation-z1" [attr.id]="operatorLabelID">
-  {{operator.additionalMetadata.userFriendlyName}}
+  {{operator?.additionalMetadata.userFriendlyName}}
```

Comment:
Angular Safe Navigation Operator: https://angular.io/guide/template-syntax#safe-navigation-operator

It's a guard to prevent errors caused by `null` or `undefined`